src/test/ui/suggestions/fn-closure-mutable-capture.nll.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0594]: cannot assign to immutable item `x`
+error[E0594]: cannot assign to `x` which is behind a `&` reference
```

Comment:
Hmm, maybe we should flag this issue as a follow-up work item — it seems a bit confusing, since the reference here is really introduced by the closure desugaring, right?